Assets/Shopify/UIToolkit/Theme Controllers/ThemeControllerBase.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace Shopify.UIToolkit {
     using UnityEngine;
+    using UnityEditor;
```

Comment:
this won't build (to a player), as the editor assembly is not available during the runtime. You should put this functionality into the BaseThemeControllerEditor and inject the correct credentials from there as EditorPrefs is only supposed to be used from the editor context.